components/raftstore/src/store/read_queue.rs

```diff
@@ -77,6 +77,20 @@ where
         }
     }
 
+    pub fn noop(id: Uuid, propose_time: Timespec) -> Self {
+        RAFT_READ_INDEX_PENDING_COUNT.inc();
+        ReadIndexRequest {
+            id,
+            cmds: MustConsumeVec::new("noop"),
```

Comment:
I believe there are codes that asserts `cmds` is not empty.

Reply:
I only found one place checks `cmds.len() == 1` if `addition_request` is not None.